src/operator/controllers/intents_reconcilers/otterizecloud/status_report.go

```diff
@@ -0,0 +1,38 @@
+package otterizecloud
+
+import (
+	"context"
+	"github.com/otterize/intents-operator/src/shared/otterizecloud/graphqlclient"
+	"github.com/otterize/intents-operator/src/shared/otterizecloud/otterizecloudclient"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+func PeriodicallyReportConnectionToCloud(client CloudClient) {
```

Comment:
Should have 'Start' in the name to indicate that it's starting a background process